src/datagen/main/java/mekanism/common/recipe/impl/CrusherRecipeProvider.java

```diff
@@ -512,4 +517,82 @@ private void addCrusherDewaxingRecipes(RecipeOutput consumer, String basePath) {
+        biofuel(consumer, basePath, "fern", 2, 1, Items.FERN);
+        biofuel(consumer, basePath, "fungus", 1, 1, Items.CRIMSON_FUNGUS, Items.WARPED_FUNGUS);
+        biofuel(consumer, basePath, "glow_lichen", 1, 1, Items.GLOW_LICHEN);
+        biofuel(consumer, basePath, "hanging_roots", 1, 1, Items.HANGING_ROOTS);
+        biofuelBlock(consumer, basePath, "hay_block", 2, 1, Items.HAY_BLOCK);
+        biofuel(consumer, basePath, "kelp", 2, 1, Items.KELP);
+        biofuel(consumer, basePath, "large_fern", 3, 1, Items.LARGE_FERN);
+        biofuel(consumer, basePath, "leaves", 1, 10, ItemTags.LEAVES);
+        biofuel(consumer, basePath, "lily_pad", 1, 1, Items.LILY_PAD);
+        biofuel(consumer, basePath, "mangrove_roots", 1, 1, Items.MANGROVE_ROOTS);
+        biofuel(consumer, basePath, "melon", 6, 1, Items.MELON);
+        biofuel(consumer, basePath, "melon_slice", 1, 1, Items.MELON_SLICE);
+        biofuel(consumer, basePath, "moss_block", 2, 1, Items.MOSS_BLOCK);
+        biofuel(consumer, basePath, "moss_carpet", 4, 3, Items.MOSS_CARPET);
+        biofuel(consumer, basePath, "mushroom_blocks", 4, 1, Items.BROWN_MUSHROOM_BLOCK, Items.RED_MUSHROOM_BLOCK, Items.MUSHROOM_STEM);
+        biofuel(consumer, basePath, "mushrooms", 1, 1, Tags.Items.MUSHROOMS);
+        biofuel(consumer, basePath, "nether_sprouts", 1, 1, Items.NETHER_SPROUTS);
+        biofuelBlock(consumer, basePath, "nether_wart_block", 2, 1, Items.NETHER_WART_BLOCK);
+        biofuel(consumer, basePath, "pitcher_pod", 1, 1, Items.PITCHER_POD);
+        final int pumpkinOutput = 6;
+        biofuel(consumer, basePath, "pumpkin", pumpkinOutput, 1, Items.PUMPKIN);
+        biofuel(consumer, basePath, "pumpkin_pie", pumpkinOutput + 1, 1, Items.PUMPKIN_PIE);
+        biofuel(consumer, basePath, "rotten_flesh", 1, 1, Items.ROTTEN_FLESH);
+        biofuel(consumer, basePath, "saplings", 1, 1, ItemTags.SAPLINGS);
+        biofuel(consumer, basePath, "sea_pickle", 1, 1, Items.SEA_PICKLE);
+        biofuel(consumer, basePath, "seagrass", 1, 1, Items.SEAGRASS);
+        biofuel(consumer, basePath, "seeds", 1, 1, Tags.Items.SEEDS);
+        biofuel(consumer, basePath, "short_grass", 1, 1, Items.SHORT_GRASS);
+        biofuel(consumer, basePath, "shroomlight", 4, 1, Items.SHROOMLIGHT);
+        biofuel(consumer, basePath, "small_dripleaf", 1, 1, Items.SMALL_DRIPLEAF);
+        biofuel(consumer, basePath, "small_flowers", 1, 1, ItemTags.SMALL_FLOWERS);
+        biofuel(consumer, basePath, "pink_petals", 1, 1, Items.PINK_PETALS);//not in small_flowers tag
+        biofuel(consumer, basePath, "spore_blossom", 2, 1, Items.SPORE_BLOSSOM);
+        biofuel(consumer, basePath, "sugar_cane", 1, 1, Items.SUGAR_CANE);
+        biofuel(consumer, basePath, "tall_flowers", 2, 1, ItemTags.TALL_FLOWERS);
+        biofuel(consumer, basePath, "tall_grass", 2, 1, Items.TALL_GRASS);
+        biofuel(consumer, basePath, "torchflower_seeds", 1, 1, Items.TORCHFLOWER_SEEDS);
```

Comment:
I believe this is redundant, as this is included in `Tags.Items.SEEDS`

Reply:
hmm it wasn't showing up when i last looked